orm/model/ormtable/table.go

```diff
@@ -17,7 +17,17 @@ import (
 // systems, for instance to enable backwards compatibility when a major
 // migration needs to be performed.
 type View interface {
-	UniqueIndex
+	Index
+
+	// Has returns true if there is an entity in the table with the same
+	// primary key as message. Other fields besides the primary key fields will not
+	// be used for retrieval.
+	Has(ctx context.Context, message proto.Message) (found bool, err error)
+
+	// Get retrieves the message if one exists for the primary key fields
+	// set on the message. Other fields besides the primary key fields will not
+	// be used for retrieval.
+	Get(ctx context.Context, message proto.Message) (found bool, err error)
```

Comment:
@fdymylja added these to the `Table` interface as this makes `Has`/`Get` usage based on the primary key simpler.